src/extra/JsonSchemaRenderer/JsonSchemaRendererPlayground/index.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from 'react';
+import { keys, get, escapeRegExp, filter } from 'lodash';
+import JsonSchema7Schema from 'ajv/lib/refs/json-schema-draft-07.json';
+import jsone from 'json-e';
+import styled from 'styled-components';
+import { px } from 'styled-system';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUndo } from '@fortawesome/free-solid-svg-icons/faUndo';
+import asRendition from '../../../asRendition';
+import {
+	DefaultProps,
+	RenditionSystemProps,
+	Theme,
+} from '../../../common-types';
+import { Flex } from '../../../components/Flex';
+import Card from '../../../components/Card';
+import Select from '../../../components/Select';
+import Button from '../../../components/Button';
+import Heading from '../../../components/Heading';
+import JsonSchemaRenderer, { JsonSchemaRendererProps } from '../index';
+import { Format, Value, JSONSchema, UiSchema } from '../types';
+import JsonEditor from './JsonEditor';
+import { generateUiSchemaMetaSchema } from './util';
+
+const EXTRA_FORMATS: Format[] = [
+	{
+		name: 'markdown',
+		format: '.*',
+	},
+];
+
+const SlimCard = styled(Card)`
```

Comment:
Generally speaking, overrides like this point to a problem with either the designs or the Card, and one of those should be fixed. The card does need a bit of work to match recent designs, but some are intentional, like the padding (you can use the `small` prop on the card for less padding). Other than that, the card should accept Flex props, and if it doesn't can you look into it? AFAIK the wrapper is a Flex so it should work just fine without all the overrides.

Reply:
The Card component is actually surprisingly unwieldy. 
* I've removed the padding override and settled for the 'small' padding (which is still pretty big IMHO).  
* I've moved the background-color override to a `backgroundColor` 
* I've fixed `Card` so that is now accepts the `flex` property
* The `Card` component contains various `Box` components (which are the divs I'm styling in my styled Cards). I don't want to change these to `Flex` components and specify flex properties on them as this would be a breaking change to Rendition. Ok to keep it like this for now? 

On a related note, I'm often replacing `Box` with `Flex` in my own code. I think we often assume a `Box` will suffice when actually a `Flex` copes with many more scenarios. Although obviously this is an over-simplification.